ovh/helpers/helpers.go

```
@@ -293,6 +310,16 @@ func GetNilIntPointer(val interface{}) *int {
 	return &value
 }
 
+func GetNilFloat64Pointer(val interface{}) (*float64, error) {
+	if val == nil {
+		return nil, nil
+	} else if value, ok := val.(float64); ok {
+		return &value, nil
+	} else {
```

same, `else` not needed, you can return the error outside of the block